5-Heroku/src/index.js

```diff
@@ -0,0 +1,215 @@
+    itemElement.classList.add(classes.todoItem)
+    itemElement.id = itemID;
+
+    const itemCheckboxElement = createItemCheckboxElement(itemID)
+    const itemTextElement = createItemTextElement(itemID, title, content)
+    const itemButtonsElement = createItemButtonsElement(itemID)
+
+    itemElement.appendChild(itemCheckboxElement)
+    itemElement.appendChild(itemTextElement)
+    itemElement.appendChild(itemButtonsElement)
+
+    return itemElement
+}
+function createItemCheckboxElement(itemID) {
+    const itemCheckboxElement = document.createElement('input')
+    itemCheckboxElement.type = 'checkbox'
+    itemCheckboxElement.classList.add(classes.itemCheckbox)
+    itemCheckboxElement.id = `itemCheckbox${itemID}`
+    itemCheckboxElement.onclick = function () {
+        const titleElement = document.getElementById(`title${itemID}`)
+        const contentElement = document.getElementById(`content${itemID}`)
+        const editButtonElement = document.getElementById(`itemEdit${itemID}`)
+        if (!this.checked) {
+            titleElement.classList.remove(classes.strike)
+            contentElement.classList.remove(classes.invisible)
+            editButtonElement.classList.remove(classes.invisible)
+        } else {
+            titleElement.classList.add(classes.strike)
+            contentElement.classList.add(classes.invisible)
+            editButtonElement.classList.add(classes.invisible)
+        }
+    }
+    return itemCheckboxElement
+}
+function createItemButtonsElement(itemID) {
+
+    const itemButtonsElement = document.createElement('div')
+    itemButtonsElement.classList.add(classes.todoItemButtons)
+
+    const editButtonElement = document.createElement('button')
+    editButtonElement.id = `itemEdit${itemID}`
+    editButtonElement.innerText = 'edit'
+    editButtonElement.classList.add(classes.itemEditButton)
+    editButtonElement.onclick = function () {
+
+        editMode = true
+        editedID = itemID
+
+        document.getElementById(constants.cancelButtonID).hidden = false
+        setInnerText(constants.applyButtonID, 'Apply')
+        const title = getInnerText(`title${itemID}`)
+        const content = getInnerText(`content${itemID}`)
+        setValue(constants.titleInputID, title)
+        setValue(constants.contentInputID, content)
+    }
+
+    const deleteButtonElement = document.createElement('button')
+    deleteButtonElement.id = `itemDelete${itemID}`
+    deleteButtonElement.innerText = 'delete'
+    deleteButtonElement.classList.add(classes.itemDeleteButton)
+    deleteButtonElement.onclick = function () {
+        if (editMode) {
+            alert('cant delete while in edit mode')
+            return
+        }
+        const item = document.getElementById(itemID)
+        const list = document.getElementById(constants.todoListID)
+
+        list.removeChild(item)
+
+        storage.remove(itemID)
+    }
+
+    itemButtonsElement.appendChild(editButtonElement)
```

Comment:
i think that u can do
`itemButtonsElement.append(editButtonElement, deleteButtonElement)`